lib/krane/cli/deploy_command.rb

```diff
@@ -24,26 +21,19 @@ class DeployCommand
                                     default: PROTECTED_NAMESPACES },
         "prune" => { type: :boolean, desc: "Enable deletion of resources that do not appear in the template dir",
                      default: true },
-        "render-erb" => { type: :boolean, desc: "Enable ERB rendering", default: false },
         "selector" => { type: :string, banner: "'label=value'",
                         desc: "Select workloads by selector(s)" },
         "verbose-log-prefix" => { type: :boolean, desc: "Add [context][namespace] to the log prefix",
                                   default: true },
         "verify-result" => { type: :boolean, default: true,
                              desc: "Verify workloads correctly deployed" },
-        "current-sha" => { type: :string, banner: "SHA", desc: "Expose SHA `current_sha` in ERB bindings" },
-
```

Comment:
As mentioned IRL, I think we should remove `current-sha` from `DeprecatedDeployTask#statsd_tags` and `ResourceWatcher#statsd_tags`, due to cardinality concerns.